Makefile

```diff
@@ -35,6 +35,7 @@ WEAVE_EXPORT=weave.tar
 WEAVEEXEC_DOCKER_VERSION=1.3.1
 DOCKER_DISTRIB=prog/weaveexec/docker-$(WEAVEEXEC_DOCKER_VERSION).tgz
 DOCKER_DISTRIB_URL=https://get.docker.com/builds/Linux/x86_64/docker-$(WEAVEEXEC_DOCKER_VERSION).tgz
+COVERAGE_MODULES=./common,./common/docker,./ipam,./ipam/paxos,./ipam/ring,./ipam/space,./net,./net/address,./router
```

Comment:
```
space:=$(subst x, ,x)
comma:=,
COVERAGE_MODULES=$(subst $(space),$(comma),$(shell find ./common ./ipam ... -type d))
```

(I'll leave implementing `find -type d` in pure make as an exercise for the reader.)